docs/src/main/asciidoc/ap4k.adoc

```diff
@@ -0,0 +1,155 @@
+= {project-name} - Deploying an Application to Kubernetes and OpenShift
```

Comment:
Maybe `using AP4K` at the end? We already have guides having similar titles. Or maybe we should revisit that to push users to use that approach?